src/lib.rs

```diff
@@ -55,7 +55,7 @@ pub trait HasContext {
     type Fence: Copy + Clone + Debug + Eq + Hash + Ord + PartialEq + PartialOrd;
     type Framebuffer: Copy + Clone + Debug + Eq + Hash + Ord + PartialEq + PartialOrd;
     type Renderbuffer: Copy + Clone + Debug + Eq + Hash + Ord + PartialEq + PartialOrd;
-    type UniformLocation: Copy + Clone + Debug + Eq + Hash + Ord + PartialEq + PartialOrd;
+    type UniformLocation: Clone + Debug;
```

Comment:
It's a bit unfortunate to lose these traits, but fortunately I don't think they're very useful for `UniformLocation` anyway

Reply:
I guess the associated type should go away entirely.